docs/source/troubleshooting.md

```diff
@@ -37,11 +37,11 @@ to the config file, `jupyterhub_config.py`.
 When nothing is given for these lists, there will be no admins, and all users
 who can authenticate on the system (i.e. all the Unix users on the server with
 a password) will be allowed to start a server. The allowed username set lets you limit
-this to a particular set of users, and admin_users lets you specify who
+this to a particular set of users, and admin_users let you specify who
```

Comment:
`admin_users` is a singular configuration value, so it should be `lets` not `let.